tools/ci_build/github/azure-pipelines/templates/orttraining-linux-gpu-test-ci-pipeline.yml

```diff
@@ -39,6 +39,7 @@ steps:
   timeoutInMinutes: 60
 
 # Entry point for all ort training api tests
+# TODO: move onnxscript installation to CI image.
```

Comment:
I am not sure when it will be the right time as onnxscript is a relatively new tool.